src/Config.ts

```diff
@@ -0,0 +1,135 @@
+import * as fs from 'fs-extra';
+import { Account } from './Account';
+import { MosaicConfig } from './MosaicConfig';
+
+// Database password key to read from env.
+const DB_PASSWORD = 'MOSAIC_FACILITATOR_DB_PASSWORD';
+
+// Database type
+enum DBType {
+  SQLITE = 'SQLITE',
+}
+
+
+/**
+ * Holds mosaic config, database config and facilitator config.
+ */
+export class Config {
+  public database: DBConfig;
+
+  public chains: Record<string, any>;
+
+  public mosaic: MosaicConfig;
+
+  /**
+   * Constructor.
+   * @param mosaicConfigPath Mosaic config path.
+   * @param facilitatorConfigPath Facilitator config path.
```

Comment:
is it not a facilitator manifest?

Reply:
Manifest to me sounds like it should contain the version, app name, package identifier. etc.
I still think that its config.
The file defines the configuration to run the facilitator.

I may be wrong here though.